pkg/ytconfig/generator.go

```diff
@@ -301,8 +301,7 @@ func (g *NodeGenerator) fillBusServer(c *CommonServer, s *ytv1.RPCTransportSpec)
 		c.BusServer = &BusServer{}
 	}
 
-	// FIXME(khlebnikov): some clients does not support TLS yet
-	if s.TLSRequired && s != g.commonSpec.NativeTransport {
```

Comment:
We are not ready to remove this kluge.
Before making TLS fully required we need at least e2e test for that.